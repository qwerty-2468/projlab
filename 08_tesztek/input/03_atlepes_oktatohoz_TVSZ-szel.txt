create room r1
create room r2
create student s
create teacher t
create tvsz tv

neighbour r1 r2
addperson s r1
addperson t r2
add tv s
enter r2 s

status tv
status