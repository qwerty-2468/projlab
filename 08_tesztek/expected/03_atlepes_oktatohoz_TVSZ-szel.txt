
tv
location: r2
holder: s
usesRemaining: 2
Game in progress
